testing/e2e/skr/skr-test/README.md

```diff
@@ -0,0 +1,89 @@
+# SKR test
+
+This test covers SKR (SAP Kyma Runtime).
+
+## File structure
+- **provision** folder contains the scripts for provisioning and de-provisioning the SKR cluster using KEB client
+- **oidc** folder contains the OIDC-related tests
+
+## Usage modes
+
+You can use the SKR test in two modes - with and without provisioning.
+
+### With provisioning
+
+In this mode, the test executes the following steps:
+
+1. Provision an SKR cluster.
+2. Run the OIDC test.
+3. Deprovision the SKR instance and clean up the resources.
+
+### Without Provisioning.
+
+In this mode the test additionally needs the following environment variables:
+- `SKIP_PROVISIONING`, set to `true`
+- `INSTANCE_ID` the uuid of the provisioned SKR instance
+
+In this mode, the test executes the following steps:
+1. Ensure the SKR exists.
+2. Run the OIDC test.
+3. Clean up the resources.
+ 
+>**NOTE:** The SKR test additionally contains a stand-alone script that you can use to register the resources.
+
+## Test execution
+
+1. Before you run the test, prepare the `.env` file based on the following `.env.template`:
```

Comment:
If there is another, exactly the same, `.env.template`, you might just link to it.